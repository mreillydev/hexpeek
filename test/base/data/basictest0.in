margin 2
cols 20
@@+
offset
0/77
offset
0 search .2
offset
/20776F
offset
@@+/ 20776F
offset
0+/ 20    77 6F
offset
7:Csearch .1
offset
@7:D/.1
offset
@4/@2
offset
4/@2,2
offset
text
0
