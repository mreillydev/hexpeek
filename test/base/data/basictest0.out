0000: 48656c6c 6f2c2077 6f726c64 2121210a
10
At 7 (20 octets requested, 20 per line, hexadecimal) :
0007: 776f726c 64212121 0a
7
At 9 (20 octets requested, 20 per line, hexadecimal) :
0009: 726c6421 21210a
9
9
10
At 6 (20 octets requested, 20 per line, hexadecimal) :
0006: 20776f72 6c642121 210a
9
7
At c (20 octets requested, 20 per line, hexadecimal) :
000c: 2121210a
c
At a (20 octets requested, 20 per line, hexadecimal) :
000a: 6c642121 210a
a
4
0000: 48656c6c 6f2c2077 6f726c64 2121210a                                      Hello, world!!!.
